docker file problem

<!--
Thanks for opening an issue! To help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. 
-->

docker build runs ok

should run ok.

does not run ok

npm info it worked if it ends with ok
npm info using npm@4.2.0
npm info using node@v7.10.0
npm ERR! Linux 4.9.27-moby
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! path /package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open

npm ERR! enoent ENOENT: no such file or directory, open '/package.json'
npm ERR! enoent ENOENT: no such file or directory, open '/package.json'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /root/.npm/_logs/2017-05-25T16_59_33_588Z-debug.log

How to reproduce: mac osx sierra 10.12.4 docker 17.03.1-ce-mac12 (17661)

<!-- For bugs, provide an URL that demos the problem -->

* Browser: 
* Operating system:

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,12 @@
-FROM node
-RUN npm i
+FROM node:alpine
+
+RUN apk add --update make g++ python
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+
+RUN yarn install
+
 EXPOSE 3000
 CMD npm start

